cli/src/test/java/uk/ac/ebi/atlas/cli/ExpressionAtlasCliApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package uk.ac.ebi.atlas.cli;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ExpressionAtlasCliApplicationTests {
+
+    @Test
+    void contextLoads() {
```

Comment:
is this supposed to be empty? if yes, whats the purpose of it? can we delete it?

Reply:
I have no idea, this is copy pasted code of something that Alfonso wrote at the other side.

Reply:
I'll check, thanks.